webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -564,7 +564,38 @@ export default class SuggestionBox extends React.PureComponent {
         return this.state.items.some((item) => !item.loading);
     };
 
+    confirmPretext = () => {
+        const textbox = this.getTextbox();
+        const pretext = textbox.value.substring(0, textbox.selectionEnd).toLowerCase();
```

Comment:
Why lower case this? I don't think we ever want the autocomplete to be case-sensitive, but looking at `handleChange` we don't actually do that normally

Reply:
Copy pasted from componentDidUpdate. Didn't think much about it.

Reply:
Huh, I don't remember that one. This seems fine to me then